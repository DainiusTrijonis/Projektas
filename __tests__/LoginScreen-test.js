/**
 * @format
 */

import 'react-native';
import React from 'react';
import LoginScreen from '../src/screens/LoginScreen';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

describe('<App rendering />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<LoginScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});