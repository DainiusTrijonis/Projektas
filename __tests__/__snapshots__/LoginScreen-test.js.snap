// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App rendering /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <TextInput
    allowFontScaling={true}
    autoCapitalize="none"
    onChangeText={[Function]}
    placeholder="Email"
    rejectResponderTermination={true}
    style={
      Object {
        "borderBottomWidth": 1,
        "borderColor": "#d3d3d3",
        "fontSize": 16,
        "margin": 10,
        "padding": 15,
        "textAlign": "center",
        "width": "85%",
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    placeholder="Password"
    rejectResponderTermination={true}
    secureTextEntry={true}
    style={
      Object {
        "borderBottomWidth": 1,
        "borderColor": "#d3d3d3",
        "fontSize": 16,
        "margin": 10,
        "padding": 15,
        "textAlign": "center",
        "width": "85%",
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <View
    accessible={true}
    focusable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#F6820D",
        "borderColor": "#F6820D",
        "borderRadius": 5,
        "borderWidth": 1,
        "marginBottom": 20,
        "marginTop": 30,
        "opacity": 1,
        "paddingVertical": 5,
        "width": 200,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#fff",
          "fontSize": 20,
          "fontWeight": "bold",
        }
      }
    >
      Login
    </Text>
  </View>
  <View
    accessibilityRole="button"
    accessibilityStates={Array []}
    accessible={true}
    focusable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Don't have an account yet? Sign up
      </Text>
    </View>
  </View>
</View>
`;
